docs/tutorial/acquiring_data/jsoc.rst

```diff
@@ -166,431 +213,457 @@ But, when the same is passed through PrimeKey attribute, it should be passed as
-    ...             a.jsoc.Segment('image'))  # doctest: +REMOTE_DATA
-    Traceback (most recent call last):
-    ...
-    ValueError: Unexpected Segments were passed. The series hmi.sharp_720s contains the following Segments ['magnetogram', 'bitmap', 'Dopplergram', 'continuum', 'inclination', 'azimuth', 'field', 'vlos_mag', 'dop_width', 'eta_0', 'damping', 'src_continuum', 'src_grad', 'alpha_mag', 'chisq', 'conv_flag', 'info_map', 'confid_map', 'inclination_err', 'azimuth_err', 'field_err', 'vlos_err', 'alpha_err', 'field_inclination_err', 'field_az_err', 'inclin_azimuth_err', 'field_alpha_err', 'inclination_alpha_err', 'azimuth_alpha_err', 'disambig', 'conf_disambig']
-
-To get files for more than 1 segment at the same time, chain ``a.jsoc.Segment()`` using ``AND`` operator::
-
-    >>> Fido.search(a.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00'),
-    ...             a.jsoc.Series('hmi.sharp_720s'),
-    ...             a.jsoc.Segment('continuum') & a.jsoc.Segment('magnetogram'))  # doctest: +REMOTE_DATA
-    <sunpy.net.fido_factory.UnifiedResponse object at ...>
-    Results from 1 Provider:
-    <BLANKLINE>
-    61 Results from the JSOCClient:
-    Source: http://jsoc.stanford.edu
-    <BLANKLINE>
-             T_REC          TELESCOP  INSTRUME WAVELNTH CAR_ROT
-    ----------------------- -------- --------- -------- -------
-    2014.01.01_00:00:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:12:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:24:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:36:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:48:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_01:00:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:00:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:12:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:24:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:36:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-                        ...      ...       ...      ...     ...
-    2014.01.01_00:24:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:36:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:48:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_01:00:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:00:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:12:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:24:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:36:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_00:48:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    2014.01.01_01:00:00_TAI  SDO/HMI HMI_SIDE1   6173.0    2145
-    Length = 61 rows
-    <BLANKLINE>
-    <BLANKLINE>
+It is necessary to specify the Segment which you need to download.
+Providing a Segment won't have any affect on the response object returned, but this will be required later, while making an export request.
```

Comment:
```suggestion
Providing a segment won't have any affect on the response object returned, but this will be required later, while making an export request.
```